<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAB FM for Sundtek Dongle</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="ls_dab">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>200</width>
      <height>360</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="ls_fm">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>200</width>
      <height>360</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>231</width>
      <height>268</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_scan">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>SCAN</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="tgl_dab_fm">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>FM</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn_tune">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>TUNE</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="btn_delete">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>60</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>DELETE</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="btn_man_tune">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>manual
Tune</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="ln_man_tune">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>25</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>MHz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_rename">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>RENAME</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="btn_add">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>ADD
STATION</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>400</y>
      <width>184</width>
      <height>92</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_kill">
       <property name="text">
        <string>kill
mediaclient</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_restart">
       <property name="text">
        <string>restart
mediaclient</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
